#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-debian.dpatch by  <nr@eecs.harvard.edu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adjust Makefile for Debian

@DPATCH@

--- mk/Makefile	2004-10-23 17:52:34.000000000 -0400
+++ mk/Makefile	2006-05-23 20:24:40.000000000 -0400
@@ -15,9 +15,9 @@
 
 VERSION = 1.6
 
-PREFIX  = /usr/local
-BINDIR  = $(PREFIX)/bin
-MAN1DIR = $(PREFIX)/man/man1
+BINDIR  = $(DESTDIR)/usr/bin
+MAN1DIR = $(DESTDIR)/usr/share/man/man1
+DOCDIR  = $(DESTDIR)/usr/share/doc/mk
 
 AR	= ar crvs
 RANLIB 	= ranlib
@@ -31,6 +31,7 @@
 install:    all
 	    install -s mk $(BINDIR)
 	    install mk.1  $(MAN1DIR)
+	    install mk.pdf $(DOCDIR)
 
 clean:
 	    rm -f *~ include/*~
